packages/terra-aggregate-translations/README.md

```diff
@@ -0,0 +1,160 @@
+
+Translations are expected to be provided in a key / value format in a JSON file. A sample translations file looks like the following:
+
+`en.json`
+
+```js
+{
+  "MyApp.alert.dismiss": "Dismiss",
+  "MyApp.alert.info": "Information.",
+  "MyApp.alert.error": "Error.",
+  "MyApp.alert.warning": "Warning.",
+  "MyApp.alert.alert": "Alert.",
+  "MyApp.alert.advisory": "Advisory.",
+  "MyApp.alert.success": "Success."
+}
+```
+
+### Order of Operations
+
+* Start with [default search patterns](https://github.com/cerner/terra-aggregate-translations/blob/main/config/defaultSearchPatterns.js)
+* Add any `custom directories` to the list of `default search patterns` to get an intermediate list of `directories to search`
+* Filter out any directories provided in the `excludes` option from the intermediate list of `directories to search`
+
+### `aggregate-translations` Options
+
+| Option | CLI Option | Type | Description | Default |
+|-|-|-|-|-|
+| baseDir | -b, --baseDir | Path | Directory to search from and to prepend to the output directory. | current working directory |
+| directories | -d, --directories | Array of Strings | Translation directory regex pattern(s) to glob, in addition to the default search patterns. | [ ] |
+| excludes | -e, --excludes | Array of Strings | Translation directory regex pattern(s) to glob exclude from the search patterns. | [ ] |
+| outputFileSystem | N/A | File System Module | The filesystem to use to write the translation and loader files. Note: The file system provide must support `mkdirp`. | [fs-extra](https://www.npmjs.com/package/fs-extra) |
+| locales  | -l, --locales | Array of Strings | The list of locale codes to aggregate. **Note: 'en' is always added if not specified.** | [terra-supported locales](https://github.com/cerner/terra-aggregate-translations/blob/main/config/i18nSupportedLocales.js) |
+| outputDir | -o, --ouputDir | String | Output directory for the translation and loader files | ./aggregated-translations |
+| configPath | -c, --config | String | The path to the terra i18n configuration file | undefined |
+| format | -f, --format | String | The format of syntax to output the translations with. Possible values are 'es5' and 'es6' | 'es5' |
+
+#### Setup Example
+
+The `aggregate-translations` setup function can be used as follows:
+
+```js
+// webpack config file
+const aggregateTranslations = require('@cerner/terra-aggregate-translations');
+
+const aggregateOptions = {
+    baseDir: __dirname,
+    excludes: ['./node_modules/packageToExclude'],
+    locales: ['en', 'en-US'],
+    format: 'es6',
+};
+
+aggregateTranslations(aggregateOptions);
+
+module.exports = // ...webpack config;
+```
+
+#### CLI Example
+
+The `aggregate-translations` CLI is supplied as a bin script, called `tt-aggregate-translations`, and can be used as follows:
+
+```js
+scripts: {
+    // ...other scripts
+    "aggregate-translations": "tt-aggregate-translations -l ['en','es'] -e ./node_modules/packageToExclude",
```

Comment:
Is there work planned to hook into terra cli?

Reply:
Logged this: https://github.com/cerner/terra-toolkit/issues/550